src/SMU-Keithley_26xx/doc/description.md

```
@@ -0,0 +1,131 @@
+# SMU Keithley 26xx
+
+This driver sources voltage or current and measures both quantities with source measure units of the
+Keithley 2600 series that use the TSP command set. One driver instance controls one channel of the
+instrument. Both channels can be used at the same time by adding two instances that share the same
+port. A pulse mode based on the `ConfigPulse` factory scripts of the instrument is available.
+
+---
+
+### Requirements:
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)